src/lib/market-service/idle/idle.ts

```diff
@@ -37,16 +38,23 @@ export class IdleMarketService extends CoinGeckoMarketService implements MarketS
 
     this.baseUrl = 'https://api.idle.finance'
     this.providerUrls = providerUrls
-    this.idle = axios.create({
-      timeout: 10000,
-      baseURL: this.baseUrl,
-      headers: {
-        Authorization:
-          'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjbGllbnRJZCI6IkFwcDIiLCJpYXQiOjE2NzAyMzc1Mjd9.pf4YYdBf_Lf6P2_oKZ5r63UMd6R44p9h5ybPprtJmT4',
+
+    this.idle = setupCache(
+      axios.create({
+        timeout: 10000,
+        baseURL: this.baseUrl,
+        headers: {
+          Authorization:
+            'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJjbGllbnRJZCI6IkFwcDIiLCJpYXQiOjE2NzAyMzc1Mjd9.pf4YYdBf_Lf6P2_oKZ5r63UMd6R44p9h5ybPprtJmT4',
+        },
+      }),
+      {
+        // 5 seconds
+        ttl: 5 * 1000,
+        interpretHeader: false,
+        staleIfError: true,
       },
-      // 5 seconds
-      cache: { maxAge: 5 * 1000, clearOnStale: true, readOnError: false, readHeaders: false },
```

Comment:
`cache` isn't available as an `Axios.create()` option anymore, so we `setupCache` instead with the matching options from `axios-cache-interceptor`